internal/pkg/runtime/engine/singularity/prepare_linux.go

```diff
@@ -530,6 +530,17 @@ func (e *EngineOperations) removeNamespace(namespaceType specs.LinuxNamespaceTyp
 	}
 }
 
+// isInNamespaces checks for existence of a specific namespace in the namespaces slice.
+func (e *EngineOperations) isInNamespaces(namespaceType specs.LinuxNamespaceType) bool {
```

Comment:
`hasNamespace` is probably a bit clearer than `isInNamespace`  because this function will be called as:

`e.hasNamespace(....)`

... and the namespace is something that the engine configuration has.